### PR TITLE
New version of CLI

### PR DESCRIPTION
**Pull request by emanuelgsouza**
The new version of Storyblok CLI comes with some changes:
* Inclusion of new task called by `sync` using the code in [storyblok-sync-script] repository (https://github.com/storyblok/storyblok-sync-script)
* Command form changes proposing the use of [commander.js](https://github.com/tj/commander.js)
* Use of [semantic-release](https://semantic-release.gitbook.io/) with [TravisCI](https://travis-ci.org/)

### CODE REVIEW

File: src/tasks/push-components.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+            console.error(e.message)
+          }
+        }
+      }
+    })
+    .catch(err => {
+      console.error(`${chalk.red('X')} An error occurred when load components file from api`)
+      return Promise.reject(err)
+    })
+  // api.get('components', (res2) => {
```
Comment (onefriendaday) [inline src/tasks/push-components.js]: Can be removed?

File: src/constants.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const LOGIN_URL = 'https://api.storyblok.com/v1/users/login'
```
Comment (onefriendaday) [inline src/constants.js]: Suggestion:
```
const API_URL = 'https://mapi.storyblok.com/v1/'
const LOGIN_URL = `${API_URL}users/login`
const SIGNUP_URL = `${API_URL}users/signup`
```

File: README.md [hunk 3]
```diff
@@ -12,42 +12,124 @@ You found an issue?<br>Tell us about it - <a href="https://github.com/storyblok/
...
+### sync
+
+Sync components, folder, roles or stories between spaces
+
+$ storyblok sync --command <COMMAND> --source <SPACE_ID> --target <SPACE_ID>
+
+#### Options
+
+* `command`: a command to execute the syncronization. Available commands:
+  * `syncFolders`
```
Comment (onefriendaday) [inline README.md]: Can we simplify this to --type folders, components, ...? Shorter to write. `storyblok sync --type folders`?

Possibility to provide multiple types would also be nice `storyblok sync --type folders components`.

File: src/constants.js (hunk 2 above)
Comment (emanuelgsouza) [reply src/constants.js]: Thanks for the suggestion, it's much better

File: README.md (hunk 3 above)
Comment (emanuelgsouza) [reply README.md]: Already implemented. Thanks for the suggestion

File: src/tasks/push-components.js (hunk 1 above)
Comment (emanuelgsouza) [reply src/tasks/push-components.js]: Already removed

**Pull request merged**